src/20_Components/amp-app-banner.html

```diff
@@ -4,7 +4,7 @@
 <!--
   #### Introduction
 
-[amp-app-banner](https://www.ampproject.org/docs/reference/components/amp-app-banner) provides a minimal UI for a cross-platform, fixed-position banner showing a call-to-action to install an app. 
+  [amp-app-banner](https://www.ampproject.org/docs/reference/components/amp-app-banner) provides a minimal UI for a cross-platform, fixed-position banner showing a call-to-action to install an app. In particular, `amp-app-banner` works inside an AMP viewer such as the Google Search News Carousel. It will fallback to existing native app banners in Safari on iOS or Chrome on Android. 
```

Comment:
What do you think about changing

"the Google Search News Carousel" to "the Top Stories carousel in Google Search"

and

"It will fallback to existing native app banners in Safari on iOS or Chrome on Android. " to "Outside of the AMP viewer, the native app banners will be displayed instead of amp-app-banner in Safari on iOS and in Chrome on Android."